CHANGELOG.md

```diff
@@ -34,6 +38,22 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [4.0.0](https://github.com/Nike-Inc/SQift/releases/tag/4.0.0)
+
+#### Added
+- Migration Guide for SQift 4 and added it to the README.
+  - Added by [Jereme Claussen](https://github.com/jereme)
+
+#### Updated
+- Deployment targets to iOS 9.0, Mac OSX 10.11, WatchOS 2.0 and tvOS 9.0.
+  - Added by [Jereme Claussen](https://github.com/jereme)
```

Comment:
`Added` -> `Updated`. Should also add the PR link the same way to do in other releases.

Reply:
Updated in 4d9a0b3